.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "*.gs": "javascript"
+      }
```

Comment:
May want to exclude this file, as it's vscode-based and is not relevant to the project.

Reply:
Removed